app/assets/javascripts/analytics/static-analytics.js

```diff
@@ -207,6 +209,26 @@
     this.setDimension(21, position);
   };
 
+  StaticAnalytics.prototype.setTotalNumberOfSections = function() {
+    var sidebarSections = $('.govuk-related-items h2').length;
```

Comment:
If you feel strongly I can remove them, but given that we're actively looking for links, I think the `a` is justified and descriptive.